.gitignore

```diff
@@ -114,3 +114,7 @@ venv.bak/
 
 # Concourse CI
 aviator_pipeline.yml
+
+# Terraform
+ci/.terraform
```

Comment:
it would probably be better to follow the pattern here:
https://github.com/dwp/aws-concourse/blob/master/.gitignore